gms/src/main/java/pl/netigen/gms/ads/AdmobInterstitial.kt

```diff
@@ -7,9 +7,11 @@ import androidx.lifecycle.LifecycleObserver
 import androidx.lifecycle.OnLifecycleEvent
 import com.google.android.gms.ads.AdListener
 import com.google.android.gms.ads.InterstitialAd
+import kotlinx.coroutines.channels.awaitClose
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.callbackFlow
 import pl.netigen.coreapi.ads.AdId
 import pl.netigen.coreapi.ads.IInterstitialAd
-import pl.netigen.coreapi.ads.InterstitialAdListener
 
 class AdmobInterstitial(
```

Comment:
IDe warns me that there's a typo in class name, I also saw it in other places. It accepts AdMob though.